deepmd/train/trainer.py

```diff
@@ -301,7 +302,10 @@ def build (self,
         else:
             log.info("training without frame parameter")
 
-        if not self.is_compress:
+        if self.is_ascend_transfer:
+            self._init_from_frz_model()
+            
```

Comment:
Thanks for the comment. The ascend transfer is very similar as training with init-model, the reason we cannot use training with init-model is the following:

1. Considering we may add new functions in the ascend transfer module in the future, developing a new module has better augmentability.
2. We cannot use train with init-model directly, since we only build a model without training. At the same time, we can automatically modify the input.json. In this way, we can finish build, freeze and transfer in one command.

Reply:
The reason we cannot use training with init-model is the following:

Considering we may add new functions in the ascend transfer module in the future, developing a new module has better augment ability.
We cannot use dp train with init-model directly, since we only build a model without training. At the same time, we can automatically modify the input.json. In this way, we can finish build, freeze and transfer in one command.